src/sentry/constants.py

```diff
@@ -37,7 +37,7 @@ def get_all_languages():
 MODULE_ROOT = os.path.dirname(__import__('sentry').__file__)
 DATA_ROOT = os.path.join(MODULE_ROOT, 'data')
 
-VERSION_LENGTH = 250
+VERSION_LENGTH = 200
```

Comment:
Due to checking the length based on the max tag length in the schema is this change still necessary/desired?